scripts/setup_thegraph_data.js

```diff
@@ -1,6 +1,6 @@
-const { solutionSubmissionParams, basicTrade } = require("../../test/resources/examples")
+const { solutionSubmissionParams, basicTrade } = require("../test/resources/examples")
 
-const { makeDeposits, placeOrders } = require("../../test/stablex/stablex_utils")
+const { makeDeposits, placeOrders } = require("../test/stablex/stablex_utils")
```

Comment:
This will probably be addressed as the tests get migrated to TS, but having `test/utilities.js` and `test/stablex/stablex_utils.js` seems redundant now.

Just mentioning it so its not forgotten, I don't know if merging the two files is in the scope of this PR.

Reply:
Will for sure merge them when I rewrite the tests in ts